scanners/flutter/flutter.go

```diff
@@ -249,74 +244,35 @@ func (scanner *Scanner) Options() (models.OptionNode, models.Warnings, models.Ic
+		var testKey string
 		if project.hasTest {
-			flutterProjectHasTestOption := models.NewOption(testsInputTitle, testsInputSummary, "", models.TypeSelector)
-			flutterProjectLocationOption.AddOption(project.path, flutterProjectHasTestOption)
-
-			for _, v := range []string{"yes", "no"} {
-				cfg := configName
-				if v == "yes" {
-					cfg += "-test"
-				}
+			testKey = "test"
+		} else {
+			testKey = "notest"
+		}
+		cfg := configName + "-" + testKey
 
-				if project.hasIosProject || project.hasAndroidProject {
-					if project.hasIosProject {
-						projectPathOption := models.NewOption(ios.ProjectPathInputTitle, ios.ProjectPathInputSummary, ios.ProjectPathInputEnvKey, models.TypeSelector)
-						flutterProjectHasTestOption.AddOption(v, projectPathOption)
+		if project.hasIosProject {
+			xcodeProjectPathOption := models.NewOption(ios.ProjectPathInputTitle, ios.ProjectPathInputSummary, ios.ProjectPathInputEnvKey, models.TypeSelector)
+			flutterProjectLocationOption.AddOption(project.path, xcodeProjectPathOption)
 
-						for xcodeWorkspacePath, schemes := range project.xcodeProjectPaths {
-							schemeOption := models.NewOption(ios.SchemeInputTitle, ios.SchemeInputSummary, ios.SchemeInputEnvKey, models.TypeSelector)
-							projectPathOption.AddOption(xcodeWorkspacePath, schemeOption)
+			for xcodeWorkspacePath, schemes := range project.xcodeProjectPaths {
+				schemeOption := models.NewOption(ios.SchemeInputTitle, ios.SchemeInputSummary, ios.SchemeInputEnvKey, models.TypeSelector)
+				xcodeProjectPathOption.AddOption(xcodeWorkspacePath, schemeOption)
 
-							for _, scheme := range schemes {
-								distributionMethodOption := models.NewOption(ios.DistributionMethodInputTitle, ios.DistributionMethodInputSummary, ios.DistributionMethodEnvKey, models.TypeSelector)
-								schemeOption.AddOption(scheme, distributionMethodOption)
+				for _, scheme := range schemes {
+					distributionMethodOption := models.NewOption(ios.DistributionMethodInputTitle, ios.DistributionMethodInputSummary, ios.DistributionMethodEnvKey, models.TypeSelector)
+					schemeOption.AddOption(scheme, distributionMethodOption)
 
-								for _, exportMethod := range ios.IosExportMethods {
-									configOption := models.NewConfigOption(cfg+"-app-"+getBuildablePlatform(project.hasAndroidProject, project.hasIosProject), nil)
-									distributionMethodOption.AddConfig(exportMethod, configOption)
-								}
-							}
-						}
-					} else {
+					for _, exportMethod := range ios.IosExportMethods {
 						configOption := models.NewConfigOption(cfg+"-app-"+getBuildablePlatform(project.hasAndroidProject, project.hasIosProject), nil)
-						flutterProjectHasTestOption.AddOption(v, configOption)
+						distributionMethodOption.AddConfig(exportMethod, configOption)
 					}
-				} else {
-					configOption := models.NewConfigOption(cfg, nil)
-					flutterProjectHasTestOption.AddOption(v, configOption)
 				}
 			}
 		} else {
-			cfg := configName
```

Comment:
This is the `!project.hasTest` path, it was exactly the same as the above code block.